oligo_designer_toolsuite/IO/_data_parser.py

```diff
@@ -234,16 +237,12 @@ def parse_frame(s):
                     for s in df[fix_quotes_column]
                 ]
             dataframes.append(df)
-    except Exception as e:
```

Comment:
Keep raising the exception and print the message inside `ParsingError`. Otherwise, the program would go on